src/components/tabs/js/tabsController.js

```diff
@@ -764,15 +765,58 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
   function refreshIndex () {
     ctrl.selectedIndex = getNearestSafeIndex(ctrl.selectedIndex);
     ctrl.focusIndex    = getNearestSafeIndex(ctrl.focusIndex);
+  } 
+
+  /**
+   * Rapid fires function to compensate for rendering lag for post-render functions.
+   * Note: Only use on inexpensive functions to avoid performace issues.
+   * @returns {*}
+   */
+  function cascadeTimeouts (func) {
+    var timeouts = [],
+        offsets = [50, 100, 300, 1000];
+
+    for (var i = 0; i < offsets.length; i++) {
+      timeouts.push($timeout(func, offsets[i]));
+    }
+
+    ctrl.scope.$on('$destroy', function () {
+      for (var i = 0; i < timeouts.length; i++) {
+        $timeout.cancel(timeouts[i]);
+      }
+    });   
+  }
+
+  /**
+   * Updates element's height of fill available space of parent element.
```

Comment:
Change `Updates element's height of fill available space` to `Updates element's height to fill available space`

Reply:
@Splaktar addressed.